source/scripting_v3/GTA/Entities/Peds/Ped.cs

```diff
@@ -862,6 +862,26 @@ public ScriptTaskStatus CurrentScriptTaskStatus
 		/// </summary>
 		public NavMeshRouteResult GetNavMeshRouteResult() => Function.Call<NavMeshRouteResult>(Hash.GET_NAVMESH_ROUTE_RESULT, Handle);
 
+		/// <summary>
+		/// This sets or unsets any VehicleChaseFlags that the <see cref="Ped"/> will use if a vehicle chase task is currently active.
+		/// You must call VehicleChase before calling this function or it will have no effect.
+		/// </summary>
+		/// <param name="flags">The VehicleChaseFlags to set or unset.</param>
+		/// <param name="set"><see langword="true"/> to set the flags; otherwise <see langword="false"/> to unset flags.</param>
+		public void VehicleChaseBehaviorFlags(VehicleChaseFlags flags, bool set)
+		{
+			Function.Call(Hash.SET_TASK_VEHICLE_CHASE_BEHAVIOR_FLAG, Handle, flags, set);
+		}
+
+		/// <summary>
+		/// This sets the distance from the target <see cref="Ped"/>'s <see cref="Vehicle"/> that the <see cref="Ped"/> will attempt to maintain if a vehicle chase task is currently active.
+		/// You must call VehicleChase before calling this function or it will have no effect.
+		/// </summary>
+		public void VehicleChaseDistance(float distance)
```

Comment:
I missed this in the first review, but Should be named like `SetVehicleChaseIdealPursuitDistance` so script devs can realize that this method would change the **ideal** pursuit distance.